tests/test_000014.textproto

```diff
@@ -0,0 +1,172 @@
+# proto-file: third_party/libiamf/cli/proto/user_metadata.proto
+# proto-message: UserMetadata
+
+test_vector_metadata {
+  human_readable_description: "A variant of `test_000005.textproto` that is encoded using Opus."
+  file_name_prefix: "test_000014"
+  is_valid: true
+  mp4_fixed_timestamp: "2023-04-19 00:00:00"
+  temporal_units_per_ia_sample: 1
+}
+
+magic_code_metadata {
+  ia_code: 0x69616d66  # "iamf"
+  version: 0
+  profile_version: 0
+}
+
+codec_config_metadata {
+  codec_config_id: 0
+  codec_config {
+    codec_id: 0x4f707573  # "Opus"
+    num_samples_per_frame: 120
```

Comment:
Are you intentionally going for the smallest frame size?

Reply:
Yes, see test_000021, test_000023, test_00024 introduced in #11 for various other frame sizes.